README.md

```diff
@@ -0,0 +1,38 @@
+# Provider
+Provider streams the iOS devices for controlfloor. This bridges the phone to the browser.
+
+# Basic Install Instructions
+* Clone repos
+git clone ios_remote_provider
```

Comment:
Generally it is good to put command that are to be run in a shell in backtics to highlight them as commands. Also; these commands cannot be run as is since they aren't links.